packages/cli/src/utils.ts

```diff
@@ -225,3 +227,40 @@ export function getTeamsAppId(rootfolder: string | undefined): any {
 
   return undefined;
 }
+
+export function getColorizedString(message: Array<{content: string, color: Colors}>): string {
+  // Only TTY support ANSI color
+  if (process.stdout.isTTY) {
+    let colorizedMessage = "";
+    (message as Array<{content: string, color: Colors}>).map(function(item) {
```

Comment:
1. The type conversion is unnecessary since `message` is already `Array<{content: string, color: Colors}>`.
2. It seems what you want to achieve is to map the array and then join the string, so why not use `message.map(...).join()` to make this more readable just like what you did in the `else` branch? The current code is [an anti-pattern on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#when_not_to_use_map)